plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineDirectManifest.kt

```diff
@@ -48,20 +51,8 @@ open class GenerateGramineDirectManifest @Inject constructor(
         // TODO We're relying on gcc, python3, pip3 and jep being installed on the machine that builds the Python
         //  enclave. Rather than documenting all this and expecting the user to have their machine correctly setup, it
         //  is better to embed the conclave-build container to always run when building the enclave, not just for
-        //  non-linux. https://r3-cev.atlassian.net/browse/CON-1181
-
-        val architecture = commandWithOutput("gcc", "-dumpmachine").trimEnd()
-        val ldPreload = executePython(
-            "from sysconfig import get_config_var; " +
-                    "print(get_config_var('LIBPL') + '/' + get_config_var('LDLIBRARY'))"
-        )
-        // The location displayed by 'pip3 show jep' is actually of the site/dist-packages dir, not the specific 'jep'
-        // dir within it. We assume this is the packages dir for other modules as well. If this assumption is
-        // incorrect then we'll need to come up with a better solution.
-        val pythonPackagesPath = commandWithOutput("pip3", "show", "jep")
-            .splitToSequence("\n")
-            .single { it.startsWith("Location: ") }
-            .substringAfter("Location: ")
+        //  non-python. https://r3-cev.atlassian.net/browse/CON-1181 and https://r3-cev.atlassian.net/browse/CON-1229.
```

Comment:
Only the python bits are left from this TODO, so update accordingly. No need for the CON-1181 reference.

Reply:
You still need python, pip and jep libraries installed on a local machine for Python enclaves. Updated accordingly.